packages/gestalt/src/Badge.js

```diff
@@ -25,6 +38,10 @@ type Props = {|
    * Text displayed inside of the Badge. Sentence case is preferred.
    */
   text: string,
+  /**
+   *  Experimental prop, do not use. Adds a [Tooltip](/web/tooltip) on hover/focus of the Badge. To convey the interaction, it also displays an information Icon. See the [type](https://gestalt.pinterest.systems/web/badge#Type) variant to learn more.
```

Comment:
If this prop is experimental, let's preface it with an underscore. That will also automatically remove it from the generated props table.

Reply:
DONE